### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by berres2002**
@maria8ch time for peer review!

### CODE REVIEW

**Comment by maria8ch** [comment]
@berres2002 finished mine as well

**Comment by berres2002** [comment]
@maria8ch 

- For 1A Background info, that latex function specifically is the PDF of the distribution, not the actual function that generates it
- Possibly improve the explanation of 1D, what distribution are you integrating that gives you a negative x-value?
- Make the statistical question in 3B more specific, like "more or as signal-like" stuff
- Non-continuous (NC) part A, add latex equation and add more information about Poisson
- NC Part B, you need to explain the features of the different plots, possibly make *more* plots
- For the Poisson plots, maybe you should explain what the histograms show since it is different from continuous distribution plots
- I think that's all.

**Comment by maria8ch** [comment]
@berres2002 

- your graph settings include `density = True`, so is y-axis really counts? I'm not sure, but you should double check.

- graphs cont: is x-axis sigmas or the range of the x-values you're drawing a distribution from? (hint- I don't think it's the sigma's)

- 1.B: again, youre not integrating to sigma, you're integrating to an x value x'

- good job on explaining z-scores!

- 1D: good job!

- Part 2, chi^2: "I wrote my own PDF function of this distribution tc2() to understand the distribution better. Where x are the x-values of the plot and k are the degrees of freedom." - reword this, it's a little hard to parse

- 2.B graphs: are the y-axis values counts? Double check this. Also, write out degrees of freedom in the title of the graph. + "make more plots" thing you wrote- either add them or remove the markdown

- Part 3: 1 and 2 of your train of thought seem to be the same. Reword this section. I know what you're saying because i did the assignment as well, but it's not quite clear to a "someone missed the lab" pov

- 3A: explain your parameters maybe? also, not sure you wanna keep your variable names such as "bruv", etc... 

- 3B: NICELY DONE!

- 3C: i'm not quite sure what you're saying here, meaning I'm not sure at all! Please clarify/ reword.

- 3D: fix typos, but nice walk-through tone

- There's a cell with a comment "going to take this out later"......  take care of it I guess?

- Part 4: for your z values, are you sure you shouldn't be doing `mydist.ppf(1 - cdf_value)` - style computation to get the accurate sigma values? (check ur math, I don't know if it's right) in the text you mention an area from positive infinity to x, but cdf goes negative infinity to x. Check that you are doing this part right, and edit the text accordingly. If this is done right, you should be good with how much you wrote.

- Discrete dists: "It is most probable that half the tests will give yes" --> "yield yes" is better wording, but I like how concise you were in this part!

- Word of warning: you had density = True everywhere else, here it isn't. Consistency is key, but if you must break consistency, be intentional about it. Just making sure you're aware.

- Nice graphs w/ varying parameters! Gets the point across well.

- This paragraph: "As stated before ..... tests with greater yesses"- well done in terms of content, just comb over the grammar for increased legibility.

- "not sure if i should remove this" cell- if you keep it, explain it.

- Part C: nicely done, maybe explain the math on this? (it's correct, just not obvious why you did it that way)

- Part D: VERY nicely done!

- Part E: " division can be a non-discrete process" - division is continuous, yes. "Unlikewise"- what in tarnation, please use a different word. "the individual tests n and the actual results for each test are discrete" - well said.

**Comment by maria8ch** [comment]
@dsharp2 Finished my Lab 1! It is up, along with an HTML rendering for you to see :)

**Comment by berres2002** [comment]
@dsharp2 finished the lab!
